### DISCUSSION TITLE
How to know if I should create an issue or a discussion?

### DISCUSSION THREAD

**Discussion by countnoobula** [laravel/framework#39249] [General]
I'm not sure where to raise this, but I noticed some of the issues were moved to discussions. 

I'm not sure how to distinguish between when I should create an issue or a discussion?
Please advise.

**Comment by SimonFrings** [comment]
@countnoobula I think there is no strict right or wrong. If you bring up a new feature idea this could be something for a discussion, but if you found a bug or something similar this should be an issue.
If you open an Issue that later evolves into an discussion, the issue gets moved (with one click).
Hope this helps :+1: 

Edit: Try open a new issue, the dialogue speaks for itself
